android/src/main/java/com/reactnativecommunity/webview/RNCWebView.java

```diff
@@ -210,6 +219,13 @@ public void callInjectedJavaScriptBeforeContentLoaded() {
         }
     }
 
+    public void injectJavaScriptObject(String obj) {
+        if (getSettings().getJavaScriptEnabled()) {
+            WebView webView = this;
+            webView.addJavascriptInterface(new JsObject(obj), "injectedObject");
```

Comment:
`injectedObject` may be a _little_ general, considering folks might not fully control the web content being loaded in their apps' WebViews.... might be worth calling it something like `ReactNativeWebViewInjectedObject` (or something shorter but specific...)

I actually wonder if it would be possible to leverage the existing `RNCWebViewBridge` instead of creating a new `JsObject` interface... then instead of `injectedObject.toString()` you could be calling `window.ReactNativeWebView.injectedObject()`...?